internal/router/router.go

```diff
@@ -347,7 +347,23 @@ func (r *Router) serveIds(w http.ResponseWriter, req *http.Request) error {
 	if key != "meta/" && key != "meta" {
 		return errgo.WithCausef(nil, params.ErrNotFound, params.ErrNotFound.Error())
 	}
-	r.Monitor.SetEndpoint("/:id/meta" + path)
+
+	// We don't want to track things that are unique to entities.
+	// For example:
+	//     /wordpress/meta/resources/foo
+	//     We want to track as /:id/meta/resources/
+	//   and
+	//     /wordpress/meta/resources
+	//     We want to track as /:id/meta/resources
```

Comment:
How about this? (the comment being added to your existing text above)

     // Technically some things after the slash might not be unique
     // to entities (for example meta/perm/read) but it's a good enough approximation
     // for now.

     if i := strings.Index(path+1, "/"); i > 0 {
         monitorPath = path[0:i+2]
     }